### PR TITLE
Add Funds support

### PR DESCRIPTION
**Pull request by asood123**
Add funds now creates a donation, which automatically processes into a transaction

### CODE REVIEW

File: migrations/20170310165323-add-notes-to-donations-migration.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+module.exports = {
+  up: function (queryInterface, Sequelize) {
+
+    return queryInterface.addColumn('Donations', 'notes', {
```
Comment (xdamman) [inline migrations/20170310165323-add-notes-to-donations-migration.js]: what's the rationale for notes in the context of a donation?
we used to use description.
we use notes as private instructions to the host to reimburse an expense

Comment (asood123) [inline migrations/20170310165323-add-notes-to-donations-migration.js]: There was only a title field on description, so if someone wanted to add more explanation, we have no way of storing that info. This allows for more details to be attached to a donation. When displaying transaction, we can pass that info on.

**Comment by asood123** [comment]
I'll merge this when frontend is ready.

**Comment by asood123** [comment]
merging to get tests to pass on website.

**Pull request merged**